lib/modules/keyboardNav.js

```diff
@@ -1183,9 +1183,15 @@ function navigateTo(newWindow, href) {
 }
 
 const goModePanel = _.once(() => {
-	const goModes = filterMap(Object.entries(module.options), ([key, opt]) => {
-		if (!opt.goMode) return;
-		return [{ key, niceKeyCode: niceKeyCode(opt.value) }];
+	const goModes = filterMap(Object.entries(module.options), ([, option]) => {
```

Comment:
Since filtering for valid shortcuts is already performed in `getActiveCommandOptions`, this may be simplified to 
```
getActiveCommandOptions()
  .filter(opt => opt.goMode)
  .map(opt => ({ key: i18n(opt.title), niceKeyCode: niceKeyCode(opt.value) }))
```

Reply:
This did not work before. Things like Legacy Profile and New Profile links showed on every page, despite only functioning on select pages.